src/playground/types.ts

```diff
@@ -2,6 +2,18 @@ import type { Diagnostic } from "@biomejs/wasm-web";
 import type { parser } from "codemirror-lang-rome-ast";
 import type { Dispatch, SetStateAction } from "react";
 
+export enum TabKeys {
```

Comment:
I would prefer if this wasn't an enum. I think it would be better if this was a union type instead.